wiki/Silence/en.md

```diff
@@ -10,17 +6,35 @@ stub: true
 
-- Edit their userpage
-- Change their avatar/profile details <!-- unconfirmed on new site, but was a thing on old -->
-- Submit [beatmaps](/wiki/Beatmap)
-- Send/receive private messages
+- Use [chat](/wiki/Chat_Console), both in-game and on the web site
+- Send and receive private messages
 - Post on any of the osu! forums
+- Comment on [beatmaps](/wiki/Beatmap)
+- Take part in [beatmap discussions](/wiki/Beatmap_Discussion)
+- Edit userpage
+- Change avatar and profile details
+- Submit and update beatmaps
+- Partake in [multiplayer](/wiki/Multi) games
+
+In addition to that, all messages of a silenced user are removed from all chat channels and private messages [1](https://blog.ppy.sh/post/38114063519/this-week-in-osu-5 "ppy blog").
+
+{{{ Infobox
+![](img/silence-profile.png "An example of silence record in profile page")
+
+![](img/silence-ingame.png "An example of silence warning shown in the chat")
+}}}
+
+After a user is silenced, they will see warning in place of the chat input field, saying when will the silence wear off. The silence record can also be observed at the bottom of the profile on the web site, from which it will disappear after a couple of weeks.
+
+## Appealing a silence
+
+If you believe you were silenced unfairly, contact [account support team](/wiki/People/The_Team/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) to contest the silence. Make sure to do it from the email linked to your osu! account; mention your osu! nickname, as well as approximate date and time when the silence was handed out.
 
 ## Common reasons for silences
 
-*Caution: Engaging in "inappropriate behaviour" in private messages is not against the rules, unless said inappropriate behaviour is being delivered to unwilling or unconsenting recipients in said private messages. If reported, appropriate punishments will be placed.*
+*Caution: Engaging in "inappropriate behaviour" in private messages is not against the rules, unless said inappropriate behaviour is being delivered to unwilling or unconsenting recipients. If reported, appropriate punishments will be placed.*
 
 - Inappropriate behaviour in public chats or forums (e.g, `#multiplayer`, beatmap discussions, comments, etc.), including—but not limited to—spamming, nonsense chat, racism, etc.
 - Uploading or posting inappropriate content via other means. (E.g., profile avatars, beatmap submissions, profile details, user page bio, banner, etc.).
 
-The length of a silence may vary based on the severity of the infringement. Silences can stack on top of each other, adding more time for each one.
+The length of a silence may vary based on the severity of the infringement. Silences can stack on top of each other, adding more time for each one. Please note that if an offense is heavy enough, or if a player is a repeat offender, it may result in [account restriction](/wiki/Help_Centre/Account_Restrictions) instead of a silence.
```

Comment:
```suggestion
The length of a silence may vary based on the severity of the infringement. Silences can stack on top of each other, adding more time for each one. Please note that if an offense is heavy enough, or if a player is a repeat offender, it may result in an [account restriction](/wiki/Help_Centre/Account_Restrictions) instead of a silence.
```

Reply:
I think this one is unnecessary

Reply:
dunno i thought it sounds more natural this way ¯\_(ツ)_/¯